operator/pkg/helm/helm.go

```diff
@@ -115,145 +82,127 @@ func renderChart(namespace, values string, chrt *chart.Chart, filterFunc Templat
 			Minor:   version.Minor,
 		}
 	}
-	vals, err := chartutil.ToRenderValues(chrt, valuesMap, options, &caps)
+	helmVals, err := chartutil.ToRenderValues(chrt, vals, options, &caps)
 	if err != nil {
-		return "", err
+		return nil, nil, fmt.Errorf("converting values: %v", err)
 	}
 
 	if filterFunc != nil {
 		filteredTemplates := []*chart.File{}
 		for _, t := range chrt.Templates {
 			// Always include required templates that do not produce any output
-			if filterFunc(t.Name) || strings.HasSuffix(t.Name, ".tpl") || t.Name == "templates/zzz_profile.yaml" || t.Name == "zzy_descope_legacy.yaml" {
+			if filterFunc(t.Name) ||
+				strings.HasSuffix(t.Name, ".tpl") ||
```

Comment:
What (otherwise presumably valid) Helm templates are we trying to explicitly exclude here, and why?

> // Always include required templates that do not produce any output

`stdout` - or what? Do we have those? Why do we need to exclude them?

I'm hopeful we won't have to add more `templates/zzz_xxx.yaml`s but if we do we might wanna come up with a naming convention that's more regexable, so we don't' have to update this every time we add new ones.

Reply:
This is stale actually, good catch. I will remove this in a commit i am working on